test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -60,4 +65,13 @@ public static Person generateTestPerson() {
             return null;
         }
     }
+
+    /**
+     * Assert whether the data in the two given files are the same.
```

Comment:
`data` --> `text`
It's better to mention that any difference in line endings are ignored.